src/text-field-base.css.ts

```diff
@@ -196,53 +283,37 @@ export const endIcon = style([
 
 export const startIcon = style([
     sprinkles({
-        paddingX: 12,
+        paddingLeft: fieldLeftPadding,
+        paddingRight: iconGap,
         display: 'flex',
         alignItems: 'center',
         height: '100%',
         position: 'absolute',
     }),
     {
-        pointerEvents: 'none', // passthrough click events to the input
+        // passthrough click events to the input
+        pointerEvents: 'none',
     },
 ]);
 
 export const prefix = style([
     sprinkles({
-        paddingLeft: 12,
+        paddingLeft: fieldLeftPadding,
         paddingRight: 16,
```

Comment:
```suggestion
        paddingRight: 12,
```

space between prefix and value is 12

I think this value may cause problems with font scaling. Use rem instead?

Just curiosity, why do you use padding instead of flexbox stuff? gap between prefix and value

Reply:
We are not using rem values for paddings that separate elements anywhere in Mística.

We could do it, but it will leave almost no space for the field's value when you increase the font size.

We can think about this a bit more in a second iteration and maybe fix this in all the components, since the problem is not happening only in input fields.

Reply:
updated padding according to specs (12 instead of 16)